### DISCUSSION TITLE
Order Retrieval using the API

### DISCUSSION THREAD

**Discussion by LittleBitsSoftware** [swellstores/community#252] [Help]
I have a situation where an order is placed (order.submitted), and a web hook event fired to my endpoint. The endpoint then retrieves the order data from Swell. The problem is that there is data missing in the response. This is custom data that has been added to the order schema. If I go back and use Postman or replay the event (raw data is captured), the custom data shows up.

Any ideas?

**Comment by maarten2424** [comment]
Hey Rob, could you send us an example of the webhook endpoint? Thanks!

**Comment by rtrainer** [comment]
This has been resolved.
